src/backends/policy_engine/souffle/sql_policy.dl

```diff
@@ -0,0 +1,49 @@
+//-----------------------------------------------------------------------------
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-----------------------------------------------------------------------------
+#ifndef SRC_BACKENDS_POLICY_ENGINE_SOUFFLE_SQL_POLICY_DL_
+#define SRC_BACKENDS_POLICY_ENGINE_SOUFFLE_SQL_POLICY_DL_
+
+#include "src/analysis/souffle/taint.dl"
+
+// OK: SELECT 42, 'foobar', null, true
+// OK: SELECT gender FROM people
+// FAIL: SELECT SSN FROM people
+//
+
+// Default owner associated with all tags related to SQL queries. 
+#define SQL_OWNER "sql"
+
+// A confidentiality tag used to capture that there restrictions.
+#define RESTRICTED_TAG "restricted"
+
+// Add restrictions on the access path.
+.decl restricted(result: AccessPath, tag: IntegrityTag)
```

Comment:
This replicates whatever `restricted` means is in the current implementation, which is the following:

> When the SQL verifier annotates something with a restricted integrity tag, we consider them to annotate that value with both a confidentiality tag and an integrity tag. This is important because an operation may mangle the IntegrityTag away while still having a result with all of its secrecy.

See the inference based on `restricted` below. The name could be changed, but just wanted to start with what already exists.